test-cases/kafka/longevity-kafka-cdc.yaml

```diff
@@ -0,0 +1,41 @@
+test_duration: 60
+pre_create_keyspace: [
+  "CREATE KEYSPACE IF NOT EXISTS keyspace1 WITH REPLICATION = {'class' : 'NetworkTopologyStrategy', 'replication_factor': 1 };",
+  "CREATE TABLE IF NOT EXISTS keyspace1.standard1 (key blob PRIMARY KEY, \"C0\" blob, \"C1\" blob, \"C2\" blob, \"C3\" blob, \"C4\" blob) WITH  cdc = {'enabled': true, 'preimage': false, 'postimage': true, 'ttl': 600}"
+]
+
+stress_cmd: ["cassandra-stress write cl=ONE n=1000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=1) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=10 -pop seq=1..10000000 -log interval=5",
+             "python_thread thread=KafakaCDCReaderThread ; read_number_of_key=1000"]
+
+n_loaders: 1
+n_monitor_nodes: 1
+n_db_nodes: 1
+
+instance_type_runner: c7i.2xlarge
+instance_type_db: i4i.large
+
+nemesis_class_name: 'NoOpMonkey'
+nemesis_interval: 1
+nemesis_filter_seeds: false
+
+user_prefix: 'kafka'
+use_mgmt: false
+
+monitor_swap_size: 0
+
+docker_network: 'kafka-stack-docker-compose_default'
+
+kafka_backend: 'localstack'
+
+kafka_connectors:
+  - version: 'hub:scylladb/scylla-cdc-source-connector:1.1.2'
```

Comment:
`version` is confusing when trying follow the code - used to install connector: how `version` can be installed - why not to split it to `source` and `version`?

Reply:
This field is used for multiple things yes,

similar to scylla_version in dtest

splitting it to multiple fields that are inter depends, it makes the config more confusing and more fields to expose in pipelines.

Reply:
changed it